lib/topological_inventory/openshift/operations/core/authentication_retriever.rb

```diff
@@ -9,10 +9,21 @@ def process
             headers = {
               "Content-Type" => "application/json"
             }
-            url = URI.join(ENV["TOPOLOGICAL_INVENTORY_URL"], "/internal/v0.0/authentications/#{@id}?expose_encrypted_attribute[]=password")
```

Comment:
Actually, where are we adding the URL in this case? Is that done somewhere else?

Reply:
Wouldn't this mean that we're calling out to `virtserver.swaggerhub.com` in production as well as tests?

Reply:
> Actually, where are we adding the URL in this case? Is that done somewhere else?

We do it globally once on startup:
https://github.com/ManageIQ/topological_inventory-collector-openshift/blob/master/bin/openshift-operations#L11-L14

> Wouldn't this mean that we're calling out to `virtserver.swaggerhub.com` in production as well as tests?

We're stubbing the requests so it isn't actually calling out to virtserver.swaggerhub.com in tests.

Reply:
Ah okay, that's exactly what I was looking for. Thanks.

> so it isn't actually calling out to virtserver.swaggerhub.com in tests.

Yeah, I got that, just making sure it wasn't configured with that URL in prod.

Reply:
:+1: